opencensus/metrics_quickstart/main.go

```diff
@@ -59,7 +59,12 @@ func main() {
 	// Exporters use Application Default Credentials to authenticate.
 	// See https://developers.google.com/identity/protocols/application-default-credentials
 	// for more details.
-	exporter, err := stackdriver.NewExporter(stackdriver.Options{})
+	exporter, err := stackdriver.NewExporter(stackdriver.Options{
+		// For demonstration the ReportingInterval is set to 10 seconds.
+		// The default value is 60 seconds. Choose the interval appropriately as it
+		// can affect the cost.
+		ReportingInterval: 10 * time.Second,
```

Comment:
I was under the impression Stackdriver's minimum reporting interval was 1 minute. Are we sure it's OK to do it even more?

Reply:
I was under the same impression until someone reported the issue with the example and they tried it with 10 seconds.  Let me check with stackdriver team.

Reply:
https://cloud.google.com/monitoring/custom-metrics/creating-metrics#writing-ts says not to call more than once per minute. Linked to from https://opencensus.io/exporters/supported-exporters/go/stackdriver/.

Reply:
given that official document refers to 60 second, I will revert the 10 second change.